src/app/tabs/test-tab.js

```diff
@@ -255,50 +280,75 @@ module.exports = class TestTab extends ViewPlugin {
     return this.runActionElement
   }
 
+  updateStopAction () {
+    const runBtn = document.getElementById('runTestsTabRunAction')
+    const stopBtn = yo`
+    <button id="runTestsTabStopAction" class="w-50 pl-2 ml-2 btn btn-secondary" title="Stop running tests" onclick=${() => this.stopTests()}">
+      <span class="fas fa-stop ml-2"></span>
+      <label class="btn btn-secondary p-1 ml-2 m-0">Stop</label>
+    </button>
+    `
+    if (runBtn && runBtn.getAttribute('disabled')) {
+      runBtn.setAttribute('disabled', 'disabled')
+    }
+    return stopBtn
+  }
+
   updateTestFileList (tests) {
     const testsMessage = (tests && tests.length ? this.listTests() : 'No test file available')
-    let el = yo`<div class=${css.testList}>${testsMessage}</div>`
+    let el = yo`<div class="${css.testList} py-2 mt-0 border-bottom">${testsMessage}</div>`
     if (!this.testFilesListElement) {
       this.testFilesListElement = el
     } else {
       yo.update(this.testFilesListElement, el)
     }
     return this.testFilesListElement
   }
+
+  selectAll () {
+    return yo`
+      <div class="d-flex mx-3 pb-2 mt-2 border-bottom">
+        <input id="checkAllTests"
+          type="checkbox"
+          onclick="${(event) => { this.checkAll(event) }}"
+          checked="true"
+        >
+        <label class="text-nowrap pl-2 mb-0" for="checkAllTests"> Select all </label>
+      </div>
+    `
+  }
+
+  infoButton () {
+    return yo`
+      <a class="btn border text-decoration-none pr-0 d-flex w-50 ml-2" title="Check out documentation." target="__blank" href="https://remix-ide.readthedocs.io/en/latest/unittesting.html">
```

Comment:
better link to redirect can be https://remix-ide.readthedocs.io/en/latest/unittesting.html#generate-test-file

Reply:
will do